.github/workflows/release_to_staging.yaml

```diff
@@ -2,6 +2,12 @@ name: Release to staging
 on:
   push:
     branches: [ main ]
+    paths:
```

Comment:
What is the idea here? Is this just for research purposes?
what if _cmd/**_ changes?